src/modules/Hosts/HostsUILib/HostsUILib.csproj

```diff
@@ -15,6 +15,10 @@
     <ProjectPriFileName>PowerToys.HostsUILib.pri</ProjectPriFileName>
     <GenerateLibraryLayout>true</GenerateLibraryLayout>
     <IsPackable>true</IsPackable>
+    <!-- AOT settings -->
+    <IsAotCompatible>true</IsAotCompatible>
```

Comment:
For these changes, I think it's fine to add the partial classes, but there's a lot more that will have to be done before it's ready to be Aot compatible, so I would hold off on making this particular project level change until prerequisites are complete (such as trimming).

Reply:
thanks for your comments! 
Just a little confused. You mean we don't need to make a particular project to AOT compatible and we need to wait for all of the PowerToys project is ready for AOT and then mark PowerToys to be AOT compatible?

Reply:
Removed AOT relate configuration.

Reply:
@snickler We've created another PR to introduce a new csproj file to enable those configuration. 
https://github.com/microsoft/PowerToys/pull/36176
In my opinion, we need to ref this new file after we can ensure a subproject is AOT compatible. 
Because we need to block another new PR if they don't match AOT requirement. If not, more and more AOT issue will be check into the master.

Reply:
Hi @snickler The current PR resolves all errors encountered after enabling IsAotCompatible. We’re considering marking the project as IsAotCompatible to prevent non-AOT code from being introduced, ensuring the project is fully AOT-compatible. At this stage, we haven’t enabled PublishAOT, so the project won’t be published as AOT yet. We plan to conduct further comprehensive testing before enabling PublishAOT. I’d appreciate any additional guidance or suggestions you might have on this approach.